### PR TITLE
Create foundation_enablement.md

### PR DESCRIPTION
**Pull request by nicholas-devlin**
### What does this PR do?
Adds a note to getting started to let people know about https://www.datadoghq.com/technical-enablement/

### Motivation
To let more people know about foundation enablement

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/getting_started/foundation_enablement.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: Datadog Learning Center
+kind: documentation
+aliases:
+  - /video-categories/
+  - /video-categories/datadog-101/
+---
+
+Datadog is the monitoring and security platform for infrastructure, applications, logs, and more. Datadog Foundation Enablement offers free live training sessions daily for customers who want to familiarize themselves with Datadog products and learn best practices across a range of observability and security use cases.
+
+To register for any of our available sessions, reference the [Enablement Portal][1].
```
Comment (urseberry) [inline content/en/getting_started/foundation_enablement.md:11]:
```suggestion
To register for available sessions, reference the [Enablement Portal][1].
```

**Review by urseberry — COMMENTED** [review]
I think it's a good idea to direct new users to the Foundation Enablement resources. However, I don't think it's large enough to warrant its own Getting Started Guide. Can you instead add a section to the existing [Learning Center guide](https://docs.datadoghq.com/getting_started/learning_center/)?

If you were to add Foundation Enablement as a new standalone guide, you would also have to update the `config/_default/menus/menus.en.yaml` file to add a new menu item.

**Comment by urseberry** [comment]
> I think it's a good idea to direct new users to the Foundation Enablement resources. However, I don't think it's large enough to warrant its own Getting Started Guide. Can you instead add a section to the existing [Learning Center guide](https://docs.datadoghq.com/getting_started/learning_center/)?
> 
> If you were to add Foundation Enablement as a new standalone guide, you would also have to update the `config/_default/menus/menus.en.yaml` file to add a new menu item.

Ah, Kaylyn has just informed me that the two of you agreed that Enablement can have its own page. I'll let her take it from here.

**Comment by apigirl** [comment]
Hey @nicholas-devlin, I am going to close this because we are in the process of adding a page into the docs that looks like the corpsite learning page that should cover this need. Thanks for thinking of it!

**Pull request closed without merging**